sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubPartitionAsyncConsumer.java

```diff
@@ -55,7 +53,8 @@
         this.consumerGroup = consumerGroup;
         this.partitionId = partitionId;
         this.trackLastEnqueuedEventProperties = trackLastEnqueuedEventProperties;
-        this.scheduler = Objects.requireNonNull(scheduler, "'scheduler' cannot be null.");
+
+        Objects.requireNonNull(scheduler, "'scheduler' cannot be null.");
```

Comment:
scheduler is not used hereafter in this class. Does it need the scheduler to pass in?